__tests__/app.test.js

```diff
@@ -70,6 +69,136 @@ describe("/api/articles", () => {
               )})
           })
       })
+      test("Status 200 - Returns array of article objects sorted by article_id", () => {
+        return request(app)
+        .get('/api/articles?sort_by=article_id')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeInstanceOf(Object);
+            expect(body).toHaveProperty('articles');
+            expect(body.articles).toBeSortedBy('article_id', {descending: true});
+            const firstArticle = body.articles[0];
+            expect(firstArticle).toEqual(
+              expect.objectContaining({
+                article_id: 12,
+                title: 'Moustache',
+                author: 'butter_bridge',
+                created_at: '2020-10-11T11:24:00.000Z',
+                topic: 'mitch',
+                votes: 0,
+                comment_count: '0'
+              }))
+            body.articles.forEach(article => {expect(article).
```

Comment:
using a forEach in a test should have an assertion that there is at least one item in the array